packages/shared-ux/page/solution_nav/src/solution_nav.tsx

```diff
@@ -110,8 +105,8 @@ export const KibanaPageTemplateSolutionNav: FunctionComponent<
   const isHidden = isLargerBreakpoint && !isOpenOnDesktop && canBeCollapsed;
   const isCustomSideNav = !!children;
 
-  const sideNavClasses = classNames('kbnPageTemplateSolutionNav', {
-    'kbnPageTemplateSolutionNav--hidden': isHidden,
+  const sideNavClasses = classNames('kbnSolutionNav', {
```

Comment:
This class name change needs to be updated here too: https://github.com/elastic/kibana/blob/2c6eb5d6d691e07126c04b390b860f2b228c3536/x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx#L69

Reply:
Nice catch, thank you!